src/components/chat/RoomList.js

```diff
@@ -18,34 +18,28 @@ const RoomList = ({
     localStorage.setItem('roomList', JSON.stringify(roomList));
   }
 
-  const styleRoomList = () => {
-    return {
-      display: 'flex',
-      flexDirection: 'column',
-      width: '100%',
-      marginTop: '48px'
-    };
-  };
-
-  const styleRoomName = (passphrase) => {
-    const lightPurple = '#aa6fe3';
-    return {
-      color: passphrase === passphraseFromUrlHash ? lightPurple : 'white',
-      width: '100%'
-    }
-  }
-
   const setPassphrase = (passphrase) => {
     console.log('setPassphrase() -- TODO: Disconnect the WebSocket, change the current URL hash, auth again, connect via WebSocket');
   }
 
+  const changeRoom = (passphrase) => {
+    // just update fragment and reload page for now
+    window.location.assign(window.location.origin + `/#${passphrase}`);
+    window.location.reload(true);
+  }
+
+  const otherRooms = roomList.filter(passphrase => passphrase !== passphraseFromUrlHash);
+
   return (
-    <div style={styleRoomList()}>
+    <div className="room-list">
       <h4>Chat Rooms</h4>
       <ul>
-        {roomList.map((passphrase) => {
+        <li className="current-room">
+          {displayRoomName(passphraseFromUrlHash)}{' '}
+        </li>
```

Comment:
@jimmcgaw Isn't it better to have a consistent room ordering, rather than moving the current room to the top?

Reply:
Seems easier to navigate, like in Slack or Discord. Yes?

Reply:
Honestly, I don't know.

Reply:
Okay I'm 90% sure leaving the same room ordering is best. Please make this tweak then let's merge?

Reply:
Done.